### ISSUE TITLE
Joining Public Network Problems

### ISSUE THREAD

**Issue by khelle** [stellar/stellar-core#1487]
Why is it so hard to join public network in Stellar Lumens? I have been running the node yestarday for the whole day, it worked fine until the block "15593577" at which it stuck. I tried to restart the node, but its still stuck on "joining SCP", not being able even to start syncing.

The "info" command gives me that:

```
[default INFO] {
   "info" : {
      "authenticated_peers_count" : 8,
      "build" : "v9.0.1-6-g226ba55",
      "ledger" : {
         "age" : 22656,
         "baseFee" : 100,
         "baseReserve" : 5000000,
         "closeTime" : 1515737722,
         "hash" : "947b681dc1cfe29ce810e5bcb9e2992055a04f917772a67ad6667acbf51b7dfb",
         "num" : 15593578,
         "version" : 9
      },
      "network" : "Public Global Stellar Network ; September 2015",
      "pending_peers_count" : 0,
      "protocol_version" : 9,
      "quorum" : {
         "15593577" : {
            "agree" : 8,
            "disagree" : 0,
            "fail_at" : 1,
            "hash" : "13b9b9",
            "missing" : 3,
            "phase" : "EXTERNALIZE"
         }
      },
      "state" : "Joining SCP"
   }
}
```

My logs are full of:

```
Jan 12 11:33:42 local stellar-core[27495]: 2018-01-12T11:33:42.851 GBA2J [Overlay INFO] New connected peer 115.87.54.253:11625
Jan 12 11:33:47 local stellar-core[27495]: 2018-01-12T11:33:47.854 GBA2J [Overlay WARNING] idle timeout
Jan 12 11:33:47 local stellar-core[27495]: 2018-01-12T11:33:47.854 GBA2J [Overlay INFO] Dropping peer GAAAA@115.87.54.253:11625
Jan 12 11:33:47 local stellar-core[27495]: 2018-01-12T11:33:47.891 GBA2J [Overlay WARNING]  connectHandler error: Operation aborted.
Jan 12 11:33:48 local stellar-core[27495]: 2018-01-12T11:33:48.862 GBA2J [Overlay INFO] New connected peer 109.66.118.89:11625
Jan 12 11:33:53 local stellar-core[27495]: 2018-01-12T11:33:53.873 GBA2J [Overlay WARNING] idle timeout
Jan 12 11:33:53 local stellar-core[27495]: 2018-01-12T11:33:53.873 GBA2J [Overlay INFO] Dropping peer GAAAA@109.66.118.89:11625
Jan 12 11:33:53 local stellar-core[27495]: 2018-01-12T11:33:53.890 GBA2J [Overlay WARNING]  connectHandler error: Operation aborted.
Jan 12 11:33:54 local stellar-core[27495]: 2018-01-12T11:33:54.887 GBA2J [Overlay INFO] New connected peer 81.171.8.70:11625
Jan 12 11:33:59 local stellar-core[27495]: 2018-01-12T11:33:59.920 GBA2J [Overlay WARNING] idle timeout
Jan 12 11:33:59 local stellar-core[27495]: 2018-01-12T11:33:59.920 GBA2J [Overlay INFO] Dropping peer GAAAA@81.171.8.70:11625
Jan 12 11:33:59 local stellar-core[27495]: 2018-01-12T11:33:59.940 GBA2J [Overlay WARNING]  connectHandler error: Operation aborted.
Jan 12 11:34:00 local stellar-core[27495]: 2018-01-12T11:34:00.904 GBA2J [Overlay INFO] New connected peer 160.152.16.202:11625
Jan 12 11:34:05 local stellar-core[27495]: 2018-01-12T11:34:05.944 GBA2J [Overlay WARNING] idle timeout
Jan 12 11:34:05 local stellar-core[27495]: 2018-01-12T11:34:05.944 GBA2J [Overlay INFO] Dropping peer GAAAA@160.152.16.202:11625
Jan 12 11:34:05 local stellar-core[27495]: 2018-01-12T11:34:05.980 GBA2J [Overlay WARNING]  connectHandler error: Operation aborted.
Jan 12 11:34:06 local stellar-core[27495]: 2018-01-12T11:34:06.912 GBA2J [Overlay INFO] New connected peer 119.39.248.75:11625
Jan 12 11:34:11 local stellar-core[27495]: 2018-01-12T11:34:11.912 GBA2J [Overlay WARNING] idle timeout
```

And nothing more than that.

My config file is as follows:

```
LOG_FILE_PATH=""

BUCKET_DIR_PATH="/home/xlm/buckets"

DATABASE="postgresql://dbname=db user=user password=pass host=127.0.0.1"


HTTP_PORT=12059

PUBLIC_HTTP_PORT=false

HTTP_MAX_CLIENT=8192

COMMANDS=[]

NODE_NAMES=[
"GAOO3LWBC4XF6VWRP5ESJ6IBHAISVJMSBTALHOQM2EZG7Q477UWA6L7U  eno",
"GCJCSMSPIWKKPR7WEPIQG63PDF7JGGEENRC33OKVBSPUDIRL6ZZ5M7OO  tempo.eu.com",
"GC5SXLNAM3C4NMGK2PXK4R34B5GNZ47FYQ24ZIBFDFOCU6D4KBN4POAE  satoshipay",
"GBJZ7PUCL4UW3NDXCYZU5Q5EUOPMBCOP2FWJA4457LYUHCQUHDM2I2K4  icici",
"GAG5HH3VF7WLUR3TRQSJGNX66GSEA5HKH3QPIYREDCT4JXUCD4U43JBJ  poiuty",
"GCGWABAQ6OUOVUGWJVPRJ5LWBIWYN3CVOVOZYBNQQGIBRULQHYNGQ7GH  cryptomover",
"GD7FVHL2KUTUYNOJFRUUDJPDRO2MAZJ5KP6EBCU6LKXHYGZDUFBNHXQI  umbrel",
"GA4UZHDP44Q6NGZ6YXA7FMTFYPZIN3XLCUMMY2KUH5FX6TGEE4SEM42K  exodo",
"GCGB2S2KGYARPVIA37HYZXVRM2YZUEXA6S33ZU5BUDC6THSB62LZSTYH  sdf_watcher1",
"GCM6QMP3DLRPTAZW2UZPCPX2LF3SXWXKPMP3GKFZBDSF3QZGV2G5QSTK  sdf_watcher2",
"GABMKJM6I25XI4K7U6XWMULOUQIQ27BCTMLS6BYYSOWKTBUXVRJSXHYQ  sdf_watcher3",
]

PEER_PORT=11625
TARGET_PEER_CONNECTIONS=8
MAX_PEER_CONNECTIONS=12
MAX_PENDING_CONNECTIONS=5000
PEER_AUTHENTICATION_TIMEOUT=5
PEER_TIMEOUT=30
PREFERRED_PEERS=[]
PREFERRED_PEER_KEYS=[]
PREFERRED_PEERS_ONLY=false
MINIMUM_IDLE_PERCENT=0
KNOWN_PEERS=[
"core-live-a.stellar.org:11625",
"core-live-b.stellar.org:11625",
"core-live-c.stellar.org:11625",
"stellar.256kw.com",
"stellar1.tempo.eu.com",
"stellar.satoshipay.io"
]


NODE_SEED="AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA self"
NODE_IS_VALIDATOR=false

FAILURE_SAFETY=1
UNSAFE_QUORUM=false


CATCHUP_COMPLETE=false
CATCHUP_RECENT=60480
MAX_CONCURRENT_SUBPROCESSES=10
MAINTENANCE_ON_STARTUP=true


RUN_STANDALONE=false
INVARIANT_CHECKS = []
MANUAL_CLOSE=false
ARTIFICIALLY_GENERATE_LOAD_FOR_TESTING=false
ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING=false
ARTIFICIALLY_SET_CLOSE_TIME_FOR_TESTING=0
ALLOW_LOCALHOST_FOR_TESTING=false

[HISTORY.local]
get="cp /home/xlm/.stellar-core/history/vs/{0} {1}"
put="cp {0} /home/xlm/.stellar-core/history/vs/{1}"
mkdir="mkdir -p /home/xlm/.stellar-core/history/vs/{0}"

[HISTORY.sdf1]
get="curl -sf http://history.stellar.org/prd/core-live/core_live_001/{0} -o {1}"

[HISTORY.sdf2]
get="curl -sf http://history.stellar.org/prd/core-live/core_live_002/{0} -o {1}"

[HISTORY.sdf3]
get="curl -sf http://history.stellar.org/prd/core-live/core_live_003/{0} -o {1}"

[QUORUM_SET]
VALIDATORS=[
"$eno", "$tempo.eu.com", "$satoshipay", "$icici", "$poiuty", "$cryptomover", "$umbrel", "$exodo", "$sdf_watcher1", "$sdf_watcher2", "$sdf_watcher3"
]
```

My system specs:
```
Ubuntu 16.04 x64
Stellar-core v9.0.1-6
SSD Hard Drive
Network Connection 100 Mbps
```

What can I do to fix this problem? Why Lumens have so much problem to connect to the PUBLIC network?

**Comment by khelle** [comment]
Its more than two hours since it started joining SCP, and didn't event start up the catch-up phase.

I would expect it to join SCP in max few minutes, and then synchronize properly, like in any other crypto currency full-node.

**Comment by MonsieurNicolas** [comment]
maybe you should start with the community that has a lot more background on this type of stuff as it seems related to the health or connectivity of some of the validators in that config file.
Based on the `info` output it looks like your node is not seeing messages from 3 validators, which is causing it to not sync.
This can be caused by those validators being down or that somehow their messages are not propagated back to your node (which can happen if they are not in other people's quorum set), so you need to put the address (ip) of some of those validators in your preferred_peers list to connect to those islands.

**Comment by khelle** [comment]
The known nodes from the config file has been copied from https://github.com/stellar/docs/blob/master/other/stellar-core-validator-example.cfg accoding to the documentation. Shouldn't the the known peers be only used for startup synchornization to init the connections but then update automatically according to the network health, so my node stays connected all the time? Do I need to manually fin which 3 of my known peers does not work and then delete them from config file? How can I do that?

**Comment by MonsieurNicolas** [comment]
You are correct that known_peers is only used for bootstrapping your list of peers.
The example config is not managed by the foundation as we do not keep track of validators: anybody can open a pull request to change it (the other similar list is the one from the dashboard).
To unblock yourself for now you can find the list of hostnames that map to the validators that you have configured (the dashboard is a good place to start as we also display if they are up) and put them as "preferred".

It's the first time I hear somebody running into this kind of problem, normally the number of connected peers is large enough to allow for redundancy. Could be triggered by latency that causes your validator to give up on nodes that are too far. Are you connected from something like AWS or home? Do you have a firewall (local or dedicated) that may kill or throttle connections to the internet?

**Comment by spadequeen7** [comment]
I am experiencing a similar problem. I failed to join the public network.
The 'info' command gives me the following:
`[default INFO] {
   "info" : {
      "authenticated_peers_count" : 8,
      "build" : "stellar-core 9.0.1 (7ad53a57f9f279d9f1697a3699ba23ed74177043)",
      "ledger" : {
         "age" : 1516071093,
         "baseFee" : 100,
         "baseReserve" : 100000000,
         "closeTime" : 0,
         "hash" : "39c2a3cd4141b2853e70d84601faa44744660334b48f3228e0309342e3f4eb48",
         "num" : 2,
         "version" : 0
      },
      "network" : "Public Global Stellar Network ; September 2015",
      "pending_peers_count" : 0,
      "protocol_version" : 9,
      "state" : "Joining SCP"
   }
}`

My logs just stuck:
`2018-01-16T02:43:45.221 GAI7K [Overlay INFO] New connected peer 46.228.6.34:11625
2018-01-16T02:43:47.221 GAI7K [Overlay WARNING] idle timeout
2018-01-16T02:43:47.221 GAI7K [Overlay INFO] Dropping peer GAAAA@46.228.6.34:11625
2018-01-16T02:43:47.221 GAI7K [Overlay WARNING]  connectHandler error: Operation aborted.
2018-01-16T02:43:48.225 GAI7K [Overlay INFO] New connected peer 18.217.85.231:11625
2018-01-16T02:43:48.265 GAI7K [Overlay WARNING] Received error (ERR_LOAD): peer rejected
2018-01-16T02:43:48.265 GAI7K [Overlay INFO] Dropping peer GBRIL@18.217.85.231:11625
2018-01-16T02:43:51.230 GAI7K [Overlay INFO] New connected peer 115.33.25.38:11625
2018-01-16T02:43:53.230 GAI7K [Overlay WARNING] idle timeout
2018-01-16T02:43:53.230 GAI7K [Overlay INFO] Dropping peer GAAAA@115.33.25.38:11625
2018-01-16T02:43:53.230 GAI7K [Overlay WARNING]  connectHandler error: Operation aborted.
2018-01-16T02:43:54.241 GAI7K [Overlay INFO] New connected peer 46.166.190.228:11625
2018-01-16T02:43:54.327 GAI7K [Overlay WARNING]  connectHandler error: Connection refused
2018-01-16T02:43:54.327 GAI7K [Overlay INFO] Dropping peer GAAAA@46.166.190.228:11625
2018-01-16T02:43:54.340 GAI7K [Overlay ERROR] TCPPeer::drop shutdown socket failed: Transport endpoint is not connected [TCPPeer.cpp:405]
2018-01-16T02:43:57.248 GAI7K [Overlay INFO] New connected peer 119.81.142.194:11625
2018-01-16T02:43:59.241 GAI7K [Overlay INFO] successful handshake with GBJ7T@119.81.142.194:11625`

And my config file is copied from 'stellar-core-validator-example.cfg' (I tried to change some of the node names, but did not work):
`LOG_FILE_PATH=""

BUCKET_DIR_PATH="/home/ubuntu/buckets"

NODE_SEED="****"
NODE_IS_VALIDATOR=true

DATABASE="postgresql://dbname=*** host=*** user=*** password=***"

FAILURE_SAFETY=1

CATCHUP_RECENT=60480

NETWORK_PASSPHRASE="Public Global Stellar Network ; September 2015"


NODE_NAMES=[
"GAOO3LWBC4XF6VWRP5ESJ6IBHAISVJMSBTALHOQM2EZG7Q477UWA6L7U  eno",
"GDZT7ZGDNEGS5L2ICLCIVDOIATKDE4MM6SNUJ2FFKB5TIV5LDRIGG4T2  lupoex",
"GC5SXLNAM3C4NMGK2PXK4R34B5GNZ47FYQ24ZIBFDFOCU6D4KBN4POAE  satoshipay",
"GB2HF2NHRKKFZYFDGD7MUENOYROOEK7SWYV2APYOODP6P7BUJTLILKIL  ibm_canada",
"GAG5HH3VF7WLUR3TRQSJGNX66GSEA5HKH3QPIYREDCT4JXUCD4U43JBJ  poiuty",
"GCGWABAQ6OUOVUGWJVPRJ5LWBIWYN3CVOVOZYBNQQGIBRULQHYNGQ7GH  cryptomover",
"GD7FVHL2KUTUYNOJFRUUDJPDRO2MAZJ5KP6EBCU6LKXHYGZDUFBNHXQI  umbrel",
"GA4UZHDP44Q6NGZ6YXA7FMTFYPZIN3XLCUMMY2KUH5FX6TGEE4SEM42K  exodo",
"GCGB2S2KGYARPVIA37HYZXVRM2YZUEXA6S33ZU5BUDC6THSB62LZSTYH  sdf_watcher1",
"GCM6QMP3DLRPTAZW2UZPCPX2LF3SXWXKPMP3GKFZBDSF3QZGV2G5QSTK  sdf_watcher2",
"GABMKJM6I25XI4K7U6XWMULOUQIQ27BCTMLS6BYYSOWKTBUXVRJSXHYQ  sdf_watcher3",
]

KNOWN_PEERS=[
"core-live-a.stellar.org:11625",
"core-live-b.stellar.org:11625",
"core-live-c.stellar.org:11625",
"stellar.256kw.com",
"ca.stellar.ibm.com",
"stellar.satoshipay.io"
]

[QUORUM_SET]
VALIDATORS=[
"$eno", "$lupoex", "$satoshipay", "$ibm_canada", "$poiuty", "$cryptomover", "$umbrel", "$exodo", "$sdf_watcher1", "$sdf_watcher2", "$sdf_watcher3"
]


# Stellar.org history store
[HISTORY.sdf1]
get="curl -sf http://history.stellar.org/prd/core-live/core_live_001/{0} -o {1}"

[HISTORY.sdf2]
get="curl -sf http://history.stellar.org/prd/core-live/core_live_002/{0} -o {1}"

[HISTORY.sdf3]
get="curl -sf http://history.stellar.org/prd/core-live/core_live_003/{0} -o {1}"
`

**Comment by MonsieurNicolas** [comment]
I am in the process of getting 9.1.0 out the door, as it has fixes for other network related problems, we'll have to re-evaluate this specific issue when validators across the network have been updated to that version

**Comment by vogel** [comment]
For fresh nodes, that do not have `peers` tables filled out, if all of their `KNOWN_PEERS` have all connection slots taken, it is impossible to connect. I think that sending list of peers with `ERR_LOAD` message would help with that issue.

**Comment by MonsieurNicolas** [comment]
It's unclear that this is related to not having enough peers in the peers table: from what I read here the node was running fine (so it should have built a pretty good list of peers) but has issues re-joining the network.

I think we need to scrub the config that is in this other repo and remove dead/unreliable validators from it. I'll try that current config file on my dev box and see what happens.

**Comment by khelle** [comment]
Ok, after few days of working on non-related tasks I came back to the stellar core. I was able to behave it little better but not perfect.

I took exactly 8 working nodes from https://dashboard.stellar.org/ (marked as "UP"), and put them in `NODE_NAMES` in my configuration file, replacing the old ones.
Then I set the same nodes as `PREFFERED_PEERS` and as `VALIDATORS`.

After all of this changes my node started connecting to the network flawessly. I let it run for random periods of 1 - 60 min, then scheduled process restart and each time it was able to reconnect. I had hoped this fixed the node behaviour but unfortunately not. I left the node for the night, came back next day to work and I saw it stuck again with this info output:

```
2018-01-26T10:59:23.089 GBA2J [default INFO] {
   "info" : {
      "authenticated_peers_count" : 12,
      "build" : "v9.0.1-6-g226ba55",
      "ledger" : {
         "age" : 23454,
         "baseFee" : 100,
         "baseReserve" : 5000000,
         "closeTime" : 1516937309,
         "hash" : "756932d9b2e8c22f104d9560d84b0abcc630ed22c785a675899ef6f79e133d38",
         "num" : 15896388,
         "version" : 9
      },
      "network" : "Public Global Stellar Network ; September 2015",
      "pending_peers_count" : 0,
      "protocol_version" : 9,
      "quorum" : {
         "15896387" : {
            "agree" : 7,
            "disagree" : 0,
            "fail_at" : 1,
            "hash" : "b3a703",
            "missing" : 3,
            "phase" : "EXTERNALIZE"
         }
      },
      "state" : "Joining SCP",
      "status" : [
         "Publishing 2203 queued checkpoints [15521727-15896383]: Succeded: prepare-snapshot"
      ]
   }
}
```

The good thing about this was that, after yesterdays changes, the restart helped. I did restart the node and it resynced without the problems.

To sum up:
- rising the number of known peers to 8 and settings them as preferred fixed the issue of node not being able to reconnect after restart
- it did NOT help in the case of node being randomly stuck

What can I do now to fix the second issue? 
Why output of "info" command still shows missing: 3 despite me replacing old known peers with different ones that dashboard sees as "Up" ?

**Comment by khelle** [comment]
@vogel @MonsieurNicolas any ideas?

**Comment by MonsieurNicolas** [comment]
There as still not enough validators running 9.1.0 on the network to investigate this further at this point.
A validator "missing" means that it's not sending any SCP message, or that its messages are being dropped by other peers (if you're not connected directly to such validator, it can be a problem).

Also "UP" on the list of validators on the dashboard just means that they are TCP pingable - not that they are actually working properly and sending SCP messages: I know some people are working on putting together something better than this as it's not enough information to pick a good validator.

**Comment by MonsieurNicolas** [comment]
most nodes on the network are running 9.2.0 now, so connectivity should be a lot better now especially when starting off fresh and the intermittent high CPU usage that could cause some nodes to drop connections.

I am closing this issue, please reopen if you are still seeing problems with 9.2.0 (and include details like you did before).

**Comment by Dinshaw7** [comment]
I am unable to sync with the public peer nodes. i started my stellar testnet node. Everything is working fine till i do new-db and then run stellar-core. After running, it does not give any error. But stops at the binding line. And does not show any connection with any peer. Hence it is stuck on seq=1. What is the issue, Can anybody help me.

**My log file-** 
2020-02-14T17:17:07.648 GDXOP [Database INFO] Connecting to: postgresql://dbname=DK user=DK
2020-02-14T17:17:07.655 GDXOP [SCP INFO] LocalNode::LocalNode@GDXOP qSet: 59d361
2020-02-14T17:17:07.760 GDXOP [default INFO] *
2020-02-14T17:17:07.760 GDXOP [default INFO] * The database has been initialized
2020-02-14T17:17:07.760 GDXOP [default INFO] *
2020-02-14T17:17:07.760 GDXOP [Database INFO] Applying DB schema upgrade to version 10
2020-02-14T17:17:07.763 GDXOP [Database INFO] Applying DB schema upgrade to version 11
2020-02-14T17:17:07.766 GDXOP [Database INFO] Applying DB schema upgrade to version 12
2020-02-14T17:17:07.770 GDXOP [Database INFO] DB schema is in current version
2020-02-14T17:17:07.771 GDXOP [Ledger INFO] Established genesis ledger, closing
2020-02-14T17:17:07.771 GDXOP [Ledger INFO] Root account seed: SDHOAMBNLGCE2MV5ZKIVZAQD3VCLGP53P3OBSBI6UN5L5XZI5TKHFQL4
2020-02-14T17:17:07.779 GDXOP [default INFO] *
2020-02-14T17:17:07.779 GDXOP [default INFO] * The next launch will catchup from the network afresh.
2020-02-14T17:17:07.779 GDXOP [default INFO] *
2020-02-14T17:17:07.779 GDXOP [default INFO] Application destructing
2020-02-14T17:17:07.779 GDXOP [default INFO] Application destroyed
2020-02-14T17:17:25.058 GDAQT [default INFO] Starting stellar-core v12.3.0-6-g18d5d5e
2020-02-14T17:17:25.059 GDAQT [Database INFO] Connecting to: postgresql://dbname=DK user=DK
2020-02-14T17:17:25.063 GDAQT [SCP INFO] LocalNode::LocalNode@GDAQT qSet: 59d361
2020-02-14T17:17:25.064 GDAQT [default INFO] Listening on 0.0.0.0:11626 for HTTP requests
2020-02-14T17:17:25.067 GDAQT [Database INFO] DB schema is in current version
2020-02-14T17:17:25.067 GDAQT [History INFO] Archive 'cache' has 'get' command only, will not be written
2020-02-14T17:17:25.067 GDAQT [History INFO] Archive 'sdf_testnet_1' has 'get' command only, will not be written
2020-02-14T17:17:25.067 GDAQT [History INFO] Archive 'sdf_testnet_2' has 'get' command only, will not be written
2020-02-14T17:17:25.067 GDAQT [History INFO] Archive 'sdf_testnet_3' has 'get' command only, will not be written
2020-02-14T17:17:25.067 GDAQT [History WARNING] No writable archives configured, history will not be written.
2020-02-14T17:17:25.067 GDAQT [Ledger INFO] Starting up application
2020-02-14T17:17:25.067 GDAQT [default INFO] Connection effective settings:
2020-02-14T17:17:25.067 GDAQT [default INFO] TARGET_PEER_CONNECTIONS: 8
2020-02-14T17:17:25.067 GDAQT [default INFO] MAX_ADDITIONAL_PEER_CONNECTIONS: 64
2020-02-14T17:17:25.067 GDAQT [default INFO] MAX_PENDING_CONNECTIONS: 500
2020-02-14T17:17:25.067 GDAQT [default INFO] MAX_OUTBOUND_PENDING_CONNECTIONS: 56
2020-02-14T17:17:25.067 GDAQT [default INFO] MAX_INBOUND_PENDING_CONNECTIONS: 444
2020-02-14T17:17:25.067 GDAQT [Ledger INFO] Last closed ledger (LCL) hash is 63d98f536ee68d1b27b5b89f23af5311b7569a24faf1403ad0b52b633b07be99
2020-02-14T17:17:25.068 GDAQT [Ledger INFO] Loaded LCL header from database: [seq=1, hash=63d98f]
2020-02-14T17:17:25.069 GDAQT [Ledger INFO] Assumed bucket-state for LCL: [seq=1, hash=63d98f]
2020-02-14T17:17:25.069 GDAQT [Overlay INFO] Binding to endpoint 0.0.0.0:11625
2020-02-14T17:18:19.722 GDAQT [default INFO] got signal 2, shutting down
2020-02-14T17:18:20.722 GDAQT [default INFO] Application destructing
2020-02-14T17:18:20.723 GDAQT [default INFO] Application destroyed
2020-02-14T17:49:09.612 GDISP [Database INFO] Connecting to: sqlite3://stellar.db
2020-02-14T17:49:09.619 GDISP [SCP INFO] LocalNode::LocalNode@GDISP qSet: 59d361
2020-02-14T17:49:09.681 GDISP [default INFO] *
2020-02-14T17:49:09.681 GDISP [default INFO] * The database has been initialized
2020-02-14T17:49:09.681 GDISP [default INFO] *
2020-02-14T17:49:09.681 GDISP [Database INFO] Applying DB schema upgrade to version 10
2020-02-14T17:49:09.684 GDISP [Database INFO] Applying DB schema upgrade to version 11
2020-02-14T17:49:09.688 GDISP [Database INFO] Applying DB schema upgrade to version 12
2020-02-14T17:49:09.690 GDISP [Database INFO] DB schema is in current version
2020-02-14T17:49:09.690 GDISP [Ledger INFO] Established genesis ledger, closing
2020-02-14T17:49:09.690 GDISP [Ledger INFO] Root account seed: SDHOAMBNLGCE2MV5ZKIVZAQD3VCLGP53P3OBSBI6UN5L5XZI5TKHFQL4
2020-02-14T17:49:09.700 GDISP [default INFO] *
2020-02-14T17:49:09.700 GDISP [default INFO] * The next launch will catchup from the network afresh.
2020-02-14T17:49:09.700 GDISP [default INFO] *
2020-02-14T17:49:09.700 GDISP [default INFO] Application destructing
2020-02-14T17:49:09.700 GDISP [default INFO] Application destroyed
2020-02-14T17:49:32.011 GCTMV [default INFO] Starting stellar-core v12.3.0-6-g18d5d5e
2020-02-14T17:49:32.012 GCTMV [Database INFO] Connecting to: sqlite3://stellar.db
2020-02-14T17:49:32.013 GCTMV [SCP INFO] LocalNode::LocalNode@GCTMV qSet: 59d361
2020-02-14T17:49:32.014 GCTMV [default INFO] Listening on 0.0.0.0:11626 for HTTP requests
2020-02-14T17:49:32.016 GCTMV [Database INFO] DB schema is in current version
2020-02-14T17:49:32.016 GCTMV [History INFO] Archive 'cache' has 'get' command only, will not be written
2020-02-14T17:49:32.016 GCTMV [History INFO] Archive 'sdf_testnet_1' has 'get' command only, will not be written
2020-02-14T17:49:32.016 GCTMV [History INFO] Archive 'sdf_testnet_2' has 'get' command only, will not be written
2020-02-14T17:49:32.016 GCTMV [History INFO] Archive 'sdf_testnet_3' has 'get' command only, will not be written
2020-02-14T17:49:32.016 GCTMV [History WARNING] No writable archives configured, history will not be written.
2020-02-14T17:49:32.016 GCTMV [Ledger INFO] Starting up application
2020-02-14T17:49:32.016 GCTMV [default INFO] Connection effective settings:
2020-02-14T17:49:32.016 GCTMV [default INFO] TARGET_PEER_CONNECTIONS: 8
2020-02-14T17:49:32.016 GCTMV [default INFO] MAX_ADDITIONAL_PEER_CONNECTIONS: 64
2020-02-14T17:49:32.016 GCTMV [default INFO] MAX_PENDING_CONNECTIONS: 500
2020-02-14T17:49:32.016 GCTMV [default INFO] MAX_OUTBOUND_PENDING_CONNECTIONS: 56
2020-02-14T17:49:32.016 GCTMV [default INFO] MAX_INBOUND_PENDING_CONNECTIONS: 444
2020-02-14T17:49:32.016 GCTMV [Ledger INFO] Last closed ledger (LCL) hash is 63d98f536ee68d1b27b5b89f23af5311b7569a24faf1403ad0b52b633b07be99
2020-02-14T17:49:32.016 GCTMV [Ledger INFO] Loaded LCL header from database: [seq=1, hash=63d98f]
2020-02-14T17:49:32.017 GCTMV [Ledger INFO] Assumed bucket-state for LCL: [seq=1, hash=63d98f]
2020-02-14T17:49:32.017 GCTMV [Overlay INFO] Binding to endpoint 0.0.0.0:11625
2020-02-14T17:54:54.575 GCTMV [default INFO] got signal 2, shutting down
2020-02-14T17:54:55.575 GCTMV [default INFO] Application destructing
2020-02-14T17:54:55.576 GCTMV [default INFO] Application destroyed
--------------------------------------------------------------------------------------------------

**My conf file-**
HTTP_PORT=11626
PUBLIC_HTTP_PORT=true
LOG_FILE_PATH="/var/log/DK/DK_stellar-core.log"

BUCKET_DIR_PATH="/var/DK/buckets"

NETWORK_PASSPHRASE="Test SDF Network ; September 2015"
#NETWORK_PASSPHRASE="Public Global Stellar Network ; September 2015"
#DATABASE="postgresql://dbname=core host=localhost user=stellar password=__PGPASS__"

DATABASE="sqlite3://stellar.db"

#DATABASE="postgresql://dbname=DK user=DK"

UNSAFE_QUORUM=true
FAILURE_SAFETY=1
CATCHUP_RECENT=8640

[HISTORY.cache]
get="cp /opt/stellar/history-cache/{0} {1}"

[[HOME_DOMAINS]]
HOME_DOMAIN="testnet.stellar.org"
QUALITY="HIGH"

[[VALIDATORS]]
NAME="sdf_testnet_1"
HOME_DOMAIN="testnet.stellar.org"
PUBLIC_KEY="GDKXE2OZMJIPOSLNA6N6F2BVCI3O777I2OOC4BV7VOYUEHYX7RTRYA7Y"
ADDRESS="core-testnet1.stellar.org"
HISTORY="curl -sf http://history.stellar.org/prd/core-testnet/core_testnet_001/{0} -o {1}"

[[VALIDATORS]]
NAME="sdf_testnet_2"
HOME_DOMAIN="testnet.stellar.org"
PUBLIC_KEY="GCUCJTIYXSOXKBSNFGNFWW5MUQ54HKRPGJUTQFJ5RQXZXNOLNXYDHRAP"
ADDRESS="core-testnet2.stellar.org"
HISTORY="curl -sf http://history.stellar.org/prd/core-testnet/core_testnet_002/{0} -o {1}"

[[VALIDATORS]]
NAME="sdf_testnet_3"
HOME_DOMAIN="testnet.stellar.org"
PUBLIC_KEY="GC2V2EFSXN6SQTWVYA5EPJPBWWIMSD2XQNKUOHGEKB535AQE2I6IXV2Z"
ADDRESS="core-testnet3.stellar.org"
HISTORY="curl -sf http://history.stellar.org/prd/core-testnet/core_testnet_003/{0} -o {1}"
-------------------------------------------------------------------------------------------------------

I tried using both postgre and sqlite.

**Output when i run stellar-core -**
[DK@uat_stellar_node ~]$ stellar-core --conf /etc/DK/DK.cfg run
2020-02-14T17:49:32.006 <startup> [default INFO] Config from /etc/DK/DK.cfg
2020-02-14T17:49:32.008 <startup> [default INFO] Generated QUORUM_SET: {
   "t" : 2,
   "v" : [ "sdf_testnet_2", "sdf_testnet_3", "sdf_testnet_1" ]
}

2020-02-14T17:49:32.011 GCTMV [default INFO] Starting stellar-core v12.3.0-6-g18d5d5e
2020-02-14T17:49:32.012 GCTMV [Database INFO] Connecting to: sqlite3://stellar.db
2020-02-14T17:49:32.013 GCTMV [SCP INFO] LocalNode::LocalNode@GCTMV qSet: 59d361
2020-02-14T17:49:32.014 GCTMV [default INFO] Listening on 0.0.0.0:11626 for HTTP requests
2020-02-14T17:49:32.016 GCTMV [Database INFO] DB schema is in current version
2020-02-14T17:49:32.016 GCTMV [History INFO] Archive 'cache' has 'get' command only, will not be written
2020-02-14T17:49:32.016 GCTMV [History INFO] Archive 'sdf_testnet_1' has 'get' command only, will not be written
2020-02-14T17:49:32.016 GCTMV [History INFO] Archive 'sdf_testnet_2' has 'get' command only, will not be written
2020-02-14T17:49:32.016 GCTMV [History INFO] Archive 'sdf_testnet_3' has 'get' command only, will not be written
2020-02-14T17:49:32.016 GCTMV [History WARNING] No writable archives configured, history will not be written.
2020-02-14T17:49:32.016 GCTMV [Ledger INFO] Starting up application
2020-02-14T17:49:32.016 GCTMV [default INFO] Connection effective settings:
2020-02-14T17:49:32.016 GCTMV [default INFO] TARGET_PEER_CONNECTIONS: 8
2020-02-14T17:49:32.016 GCTMV [default INFO] MAX_ADDITIONAL_PEER_CONNECTIONS: 64
2020-02-14T17:49:32.016 GCTMV [default INFO] MAX_PENDING_CONNECTIONS: 500
2020-02-14T17:49:32.016 GCTMV [default INFO] MAX_OUTBOUND_PENDING_CONNECTIONS: 56
2020-02-14T17:49:32.016 GCTMV [default INFO] MAX_INBOUND_PENDING_CONNECTIONS: 444
2020-02-14T17:49:32.016 GCTMV [Ledger INFO] Last closed ledger (LCL) hash is 63d98f536ee68d1b27b5b89f23af5311b7569a24faf1403ad0b52b633b07be99
2020-02-14T17:49:32.016 GCTMV [Ledger INFO] Loaded LCL header from database: [seq=1, hash=63d98f]
2020-02-14T17:49:32.017 GCTMV [Ledger INFO] Assumed bucket-state for LCL: [seq=1, hash=63d98f]
2020-02-14T17:49:32.017 GCTMV [Overlay INFO] Binding to endpoint 0.0.0.0:11625
2020-02-14T17:54:54.575 GCTMV [default INFO] got signal 2, shutting down
2020-02-14T17:54:55.575 GCTMV [default INFO] Application destructing
2020-02-14T17:54:55.576 GCTMV [default INFO] Application destroyed
---------------------------------------------------------------------------------
I waited a lot after this line - Binding to endpoint 0.0.0.0:11625, but got nothing. So destroyed application myself.

**My stellar-core version**
v12.3.0-6-g18d5d5e

**Machine**- CentOS 7
